episodes/11/assets/cluster-api/cluster.yaml

```diff
@@ -10,6 +10,10 @@ spec:
     apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
     kind: AWSCluster
     name: capi-quickstart
+  controlPlaneRef:
```

Comment:
In v1alpha3 this controlPlaneRef and the associated resources below replace the need for the separate Machine-based definitions that were needed previously in v1alpha2.

Reply:
Sounds good :) I'll take your word for it!